source/pages/uv-dose.ipynb

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
<p>Would be neat to add the parameter name and units to the x- and y-labels. E.g., <code>UV (PSM3) [W/m^2]</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>